src/transaction_history.rs

```diff
@@ -182,7 +213,7 @@ impl StoredTxAction {
         }
     }
 
-    fn into_humanized<A: Api>(self, api: &A) -> StdResult<TxAction> {
+    fn into_humanized(self) -> StdResult<TxAction> {
```

Comment:
Do we need to use this function given that we store everything as `Addr` and not `Canonical`?

Reply:
Yes it's also converting to the right TxAction

Reply:
Ok so
1) Let's change the function name to `into_tx_action`
2) Do we even need the distinction between `TxAction` and `StoredTxAction`? We might did it for legacy reasons, we might not need it anymore

Reply:
1. ok
2. i'll verify and change if possible